COLLABORATOR_GUIDE.md

```diff
@@ -263,6 +263,20 @@ multiple commits. Commit metadata and the reason for the revert should be
 appended. Commit message rules about line length and subsystem can be ignored.
 A Pull Request should be raised and approved like any other change.
 
+### Introducing New Modules
+
+Semver-minor commits that introduce new core modules should be treated with
+extra care.
+
+* First, the name of the new core module should not conflict with any existing
+  module in the ecosystem unless an agreement with the owner of those modules
+  is reached to transfer ownership.
+* PRs that introduce new core modules must be landed as semver-minor commits.
+* New core modules must be landed initially as `Experimental` and must go
+  through at least one release cycle before being moved out of experimental.
+* It is recommended to give PRs introducing new core modules more time for
+  review than the typical 48/72 hour review used for other commits.
```

Comment:
Suggest a concrete time framer here. I'd go with this:

> * Pull requests introducing new core modules should remain open for one week rather than the usual 48-hour/72-hour review period for other pull requests.

Reply:
I would suggest longer.... maybe even up to a month?

Reply:
Maybe phrase it as a minimum? (`open for at least one week`)

Reply:
I actually agree with @jasnell as this would probably have already prevented some trouble. Landing a new module should normally not be urgent.
One alternative might also be smth like `At least one week without anyone requesting any further changes.`?
And I would not make it a recommendation but mandatory with a optional fast path through a decision of the CTC/TSC?

Reply:
I think it should be possible for collaborators to pick an appropriate timeframe, even without a fixed rule. One week as a minimum sounds good, but in any case the gist of it will be “if your PR is large/has profound impact, wait until people have had a fair chance to review & assess it”

Reply:
Please explain why we don't want to go the EPS route. If a new module isn't a major addition I don't know what is.

Reply:
EPS is an ok for speculative large changes but it's really hasn't been that successful of a process over all. Having a concrete pr available, with real code, real tests, real documentation is far easier to review and look at to determine if this is something we want. For instance, Bradley's ESM pr has been a thousand times for valuable than the endless noisy threads in the EPS repo on it. Same goes for the efforts that led to N-API, URL, Inspector, etc. Yes, some discussion in EPS can be helpful but gating whether or not a semver minor can land based on that seems far too unnecessarily heavyweight of a process and, frankly, discourages innovation significantly. I'd definitely be -1 to requiring EPS and have, on several occasions, even considered proposing removing it.

Reply:
@addaleax ... "at least one week" sounds good to me but I would point out that the issue that led to me creating this issue in the first place (Jeremiah's concern that I landed the perf_hooks stuff too fast) meets all of the criteria discussed so far in this thread so I'm not sure if it's sufficient to make everyone comfortable. There needs to be a good balance so that a good compromise can be reached.